dialects/clos/phase/clos_conv.cpp

```diff
@@ -284,19 +283,21 @@ ClosConv::Stub ClosConv::make_stub(const DefSet& fvs, Lam* old_lam, Def2Def& sub
     auto num_fvs     = fvs.size();
     auto env_type    = rewrite(env->type(), subst);
     auto new_fn_type = type_clos(old_lam->type(), subst, env_type)->as<Pi>();
-    auto new_lam     = old_lam->stub(w, new_fn_type, w.dbg(old_lam->name()));
-    new_lam->set_debug_name((old_lam->is_external() || !old_lam->is_set()) ? "cc_" + old_lam->name() : old_lam->name());
+    auto new_lam     = old_lam->stub(w, new_fn_type);
+    // TODO
+    // new_lam->set_debug_name((old_lam->is_external() || !old_lam->is_set()) ? "cc_" + old_lam->name() :
+    // old_lam->name());
```

Comment:
Why not just set the name as before?
Is there a problem with the name or is it simple not necessary?